### ISSUE TITLE
prohibited-package false positive when +home URL has a trailing slash

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1379]
## Problem

A `+home` URL with a trailing slash (`https://github.com/objectionary/eo/`) causes false positives in `prohibited-package` (and breaks `mandatory-package`), because the home-repo comparison is an exact string match.

## Root cause

`src/main/resources/org/eolang/funcs/home-object.xsl:27`:

```xml
<xsl:if test="... and $tail = $eo:home-repo and ...">
```

`eo:home-object` returns `true` only when the `+home` tail equals `$eo:home-repo` exactly. `$eo:home-repo` is built without a trailing slash, so a legitimate `+home https://github.com/objectionary/eo/` (with slash) does not match → `prohibited-package.xsl:17` treats an ordinary runtime object as prohibited.

## Minimal example

```eo
+package x
+architect yegor@256.com
+version 0.0.1
+home https://github.com/objectionary/eo/
+spdx SPDX-FileCopyrightText: Copyright (c) 2026 Objectionary.com
+spdx SPDX-License-Identifier: MIT

[] > main
```

An ordinary object is flagged by `prohibited-package` because the trailing slash breaks the exact match.

## Expected behavior

Normalize the home URL (strip a trailing `/`) or use `starts-with`, so that URLs with and without a trailing slash behave identically.

## Related

- `metas/prohibited-package.xsl:17`
- `metas/mandatory-package.xsl` — affected by the same comparison.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+770](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
